tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -154,14 +154,14 @@ func (i InstantiateCCUIObject) createInstantiateCCObjects(orgNames []string, cha
 	var instantiateCCObjects []InstantiateCCUIObject
 	targetPeers := strings.Split(ccObject.TargetPeers, ",")
 	i = InstantiateCCUIObject{
-		SDK:             ccObject.SDK,
-		TransType:       action,
-		TLS:             tls,
-		ConnProfilePath: paths.GetConnProfilePath(orgNames, organizations),
-		ChainCodeID:     ccObject.ChainCodeName,
-		ChainCodeVer:    ccObject.ChainCodeVersion,
-		TargetPeers:     targetPeers,
-		Sequence:        ccObject.Sequence,
+		SDK:              ccObject.SDK,
+		TransType:        action,
+		TLS:              tls,
+		ConnProfilePaths: organizations,
```

Comment:
Field name "ConnProfilePaths" doesn't match data type "organizations".

Also, although you didn't name the function parameters, the name "organizations" leaves me wondering which organizations are these? endorsingOrganizations?

Finally didn't the legacy "instantiate" function target a single organization? Or is the function name outdated and this is really doing more than legacy instantiate?

Reply:
organizations is horrible, it's actually a list of connection profile paths for each organization, but the top level definition is organizations. It was originally connProfilePath as it only held one which didn't really work for me, I can rename that property to be something more descriptive. 
You can tell this has grown from a 1.4 instantiate to a 2.0 instantiate as it probably used to use the node sdk then moved over to using the cli and then moved to the new lifecycle but in the way I was using it for testing the network I wanted it didn't quite work so I've tweaked it to still work ok for the current stuff and should work for the new.